api/src/test/java/org/openmrs/api/impl/ConceptServiceImplTest.java

```diff
@@ -52,6 +52,8 @@
 public class ConceptServiceImplTest extends BaseContextSensitiveTest {
 	
 	protected ConceptService conceptService = null;
+
+	private static final String STANDARD_TEST_DATASET_XML = "org/openmrs/include/standardTestDataset.xml";
```

Comment:
remove this as well

Reply:
I'll resolve this squashing all commits into 1 :)